picard/ui/mainwindow.py

```diff
@@ -799,8 +799,12 @@ def play_file(self):
 
     def open_folder(self):
         files = self.tagger.get_files_from_objects(self.selected_objects)
+        opened = set()
         for file in files:
-            QtGui.QDesktopServices.openUrl(self._openUrl(os.path.dirname(file.filename)))
+            url = self._openUrl(os.path.dirname(file.filename))
+            if not url in opened:
+                QtGui.QDesktopServices.openUrl(url)
+                opened.add(url)
```

Comment:
Did you actually test this code ?

This doesn't work, because `url` is an object, so each set element is considered different.

I propose the following code to solve this issue:

```python
    def open_folder(self):
        files = self.tagger.get_files_from_objects(self.selected_objects)
        folders = set([os.path.realpath(os.path.dirname(f.filename)) for f in files])
        for folder in folders:
            QtGui.QDesktopServices.openUrl(self._openUrl(folder))
```

Advantages:
- it works
- it handles symlinked directories case
- it calls self._openUrl() once per dir